tensorflow/python/training/monitored_session.py

```diff
@@ -278,7 +278,8 @@ def MonitoredTrainingSession(master='',  # pylint: disable=invalid-name
                              scaffold=None,
                              hooks=None,
                              chief_only_hooks=None,
-                             save_checkpoint_secs=600,
+                             save_checkpoint_secs=USE_DEFAULT,
```

Comment:
cannot change the order of arguments due to backward compatibility. please add it to the end.